### DISCUSSION TITLE
take possession of current

### DISCUSSION THREAD

**Discussion by s-monzon** [neutrinolabs/xrdp#2865] [Q&A]
I am using the xrdp project, testing all the fabulous features it has. Note that it is not possible to log in via rdp without logging out locally or vice versa, causing you to lose everything you have on the desktop.
I made the configurations in sesman.ini in the line "killDisconnected=true" and "DisconnectedTimeLimit=0" but this config crashes the section which causes me to lose the work or laboratories that I am running

What solution can be given?

thank you very much

**Comment by matt335672** [comment]
@s-monzon - I'm not totally clear what you are asking.

If you want to connect to your console desktop over RDP you need to run x11vnc and use xrdp to connect through to that. Alternatively, if you're using GNOME, use GNOME remote desktop and dispense with xrdp entirely.

If you're just trying to log in over xrdp to a machine which is already logged on at the console you'll need to set up a separate user for systemd-based distros. See the [FAQ](https://github.com/neutrinolabs/xrdp/wiki/Tips-and-FAQ#why-cant-i-log-the-same-user-on-on-the-graphical-console-and-over-xrdp-at-the-same-time)
